aten/src/ATen/native/native_functions.yaml

```diff
@@ -2594,6 +2594,10 @@
   dispatch:
     QuantizedCPU: int_repr_quant
 
+- func: _per_tensor_affine_qtensor(Tensor self, float scale, int zero_point) -> Tensor
```

Comment:
I don't get the name of this function.

Reply:
It means: create a per tensor affine quantized tensor